src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1638,28 +1639,14 @@ export class ResultsViewPageStore {
         }
     });
 
-    //CBio count
-    private mutationCountByPositionKey(obj:{entrezGeneId:number, proteinPosStart:number, proteinPosEnd:number}) {
-        return `${obj.entrezGeneId}_${obj.proteinPosStart}_${obj.proteinPosEnd}`;
-    }
-
     readonly cbioportalMutationCountData = remoteData<MutationCountByPosition[]>({
         await: ()=>[
             this.mutations
         ],
         invoke: ()=>{
-            const mutationPositionIdentifiers:any = {};
-            for (const mutation of this.mutations.result!) {
-                const simplifiedMutationType = getSimplifiedMutationType(mutation.mutationType);
-                if (simplifiedMutationType === "missense" || simplifiedMutationType === "inframe") {
-                    const key = this.mutationCountByPositionKey(mutation);
-                    mutationPositionIdentifiers[key] = {
-                        entrezGeneId: mutation.entrezGeneId,
-                        proteinPosStart: mutation.proteinPosStart,
-                        proteinPosEnd: mutation.proteinPosEnd
-                    };
-                }
-            }
+
+            const mutationPositionIdentifiers = countMutations(this.mutations.result!);
```

Comment:
a good rule might be, there shall be no "logic" inside of a mobxStore invoke function.   it must use a pure helper function located in another file.   then there's question of what counts as logic.  call to _.values at 1651 is not "logic"